tck/src/main/java/org/reactivestreams/tck/IdentityProcessorVerification.java

```diff
@@ -738,6 +774,20 @@ public void expectNextElement(ManualSubscriber<T> sub, T expected) throws Interr
       }
     }
 
+    public void expectAnyNextElement(ManualSubscriber<T> sub, Collection<T> expected) throws InterruptedException {
```

Comment:
@akarnokd I disagree, the semantics of expected is that of a Set, which is why Set is the most general type with the correct semantics, instead of requiring a HashSet or a TreeSet etc.